miio/gateway.py

```diff
@@ -202,140 +189,160 @@ def set_developer_key(self, key):
     @command()
     def timezone(self):
         """Get current timezone."""
-        return self.send("get_device_prop", ["lumi.0", "tzone_sec"])
+        return self.get_gateway_prop("tzone_sec")
 
     @command()
     def get_illumination(self):
         """Get illumination. In lux?"""
-        return self.send("get_illumination")[0]
+        return self.send("get_illumination").pop()
 
 
 class GatewayAlarm(Device):
     """Class representing the Xiaomi Gateway Alarm."""
-
-    def __init__(self, parent) -> None:
-        self._device = parent
+    
+    def __init__(
+        self,
+        parent: Gateway = None,
```

Comment:
`parent` should be the last parameter (this way the base signature doesn't change).

Reply:
done